client/my-sites/themes/controller.jsx

```diff
@@ -55,6 +55,11 @@ function getProps( context ) {
 }
 
 export function upload( context, next ) {
+	if ( startsWith( context.prevPath, '/design' ) &&
```

Comment:
Maybe a short comment here? At least I feel like it is needed. I actually never know if I should comment or rely on Blame and backtracking a PR.